CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ endif()
 
 # Library type and linkage settings
 if( NOT DEFINED BUILD_FAT_LIBROCKCOMPILER )
-  set(BUILD_FAT_LIBROCKCOMPILER OFF CACHE BOOL "Build fat libMLIRMIOpen to link into MIOpen driver")
+  set(BUILD_FAT_LIBROCKCOMPILER OFF CACHE BOOL "Build fat libMLIRRock to link into Rock driver")
```

Comment:
I think my previous review is lost somehow.
The message here should be "Build fat librockCompiler"